ui/src/main/webapp/components/replications/AddReplication.js

```diff
@@ -286,6 +296,8 @@ class AddReplication extends React.Component {
                   error.graphQLErrors.forEach(e => {
                     if (e.message === 'INVALID_FILTER') {
                       this.handleInvalidFilter()
+                    } else if (e.message === 'DUPLICATE_CONFIGURATION') {
+                      this.handleInvalidName()
```

Comment:
❓ I think Chris' PR had errors for having the same source and destination, and for trying to use a site that hasn't been created yet. Does the UI have a way of handling those situations?

Reply:
The UI will restrict the sites that can be chosen to be only what is configured. It also makes sure the same site cannot be selected for both source and destination.